spec/suites/layer/PopupSpec.js

```diff
@@ -229,6 +229,18 @@ describe('Popup', function () {
 		L.Icon.Default.prototype.options.popupAnchor = popupAnchorBefore;
 	});
 
+	it("prevents an underlying map click for Layer", function () {
```

Comment:
Okay, please check. I was able to avoid adding another test in a way that still turned out pretty clean.